docs/pages/web/popover.js

```diff
@@ -617,11 +622,13 @@ function PopoverExample() {
       {open && (
         <Layer>
           <Popover
+            accessibilityLabel="Save to board"
             anchor={anchorRef.current}
             id="example-a11y"
             idealDirection="down"
             onDismiss={() => setOpen(false)}
             positionRelativeToAnchor={false}
+            role="listbox"
```

Comment:
nit: above, "For role attribute, use…"? (I can't comment on that line because GH 🙄)
- backticks around "role"
- "the" before "role"
- change "attribute" to "prop"